hw/ip/rstmgr/doc/checklist.md

```diff
@@ -0,0 +1,235 @@
+---
+title: "RSTMGR Checklist"
+---
+
+This checklist is for [Hardware Stage]({{< relref "/doc/project/development_stages.md" >}}) transitions for the [RSTMGR peripheral.]({{< relref "hw/ip/rstmgr/doc" >}})
+All checklist items refer to the content in the [Checklist.]({{< relref "/doc/project/checklist.md" >}})
+
+## Design Checklist
+
+### D1
+
+Type          | Item                  | Resolution  | Note/Collaterals
+--------------|-----------------------|-------------|------------------
+Documentation | [SPEC_COMPLETE][]     | Done        | [RSTMGR Design Spec]({{<relref "hw/ip/rstmgr/doc" >}})
+Documentation | [CSR_DEFINED][]       | Done        |
+RTL           | [CLKRST_CONNECTED][]  | Done        |
+RTL           | [IP_TOP][]            | Done        |
+RTL           | [IP_INSTANTIABLE][]   | Done        |
+RTL           | [MEM_INSTANCED_80][]  | Done        |
```

Comment:
nit: perhaps N.A. to indicate it does not have any memories?

Reply:
o yeah i can do that.